scripts/synthetix/facilitate_trade.js

```diff
@@ -51,6 +51,26 @@ const gasStationURL = {
   4: "https://safe-relay.rinkeby.gnosis.io/api/v1/gas-station/",
 }
 
+const estimationURLPrexix = {
+  1: "https://dex-price-estimator.gnosis.io//api/v1/",
+  4: "https://dex-price-estimator.rinkeby.gnosis.io//api/v1/",
+}
+
+const estimatePrice = async function (buyTokenId, sellTokenId, sellAmount, networkId) {
+  const searchCriteria = `markets/${buyTokenId}-${sellTokenId}/estimated-buy-amount/${sellAmount}?atoms=true`
+  const estimationData = await (await fetch(estimationURLPrexix[networkId] + searchCriteria)).json()
+
+  return estimationData.buyAmountInBase / estimationData.sellAmountInQuote
+}
+
+const MIN_SELL_USD = 10
+
+/* All prices refered to hear are for sETH in sUSD.
+ * The term "our" referres to prices we are buying and selling for while
+ * the term "their" refers to the buy and sell prices offered by the exchange.
```

Comment:
Can you be explicit about what is Gnosis Protocol and what is the SNX Exchange?

Reply:
and fix the typo in the word "refers". Yes!